deepchem/utils/coordinate_box_utils.py

```diff
@@ -350,3 +191,176 @@ def contains(self, other):
     return (self_x_min <= other_x_min and other_x_max <= self_x_max and
             self_y_min <= other_y_min and other_y_max <= self_y_max and
             self_z_min <= other_z_min and other_z_max <= self_z_max)
+
+
+def intersect_interval(interval1: Tuple[int, int],
+                       interval2: Tuple[int, int]) -> Tuple[int, int]:
+  """Computes the intersection of two intervals.
+
+  Parameters
+  ----------
+  interval1: Tuple[int]
```

Comment:
This type annotation is correct ...?
When seeing the `test_coordinate_box_utils.py `,  these arguments are float...
https://github.com/deepchem/deepchem/blob/ffad1a75997bb79360751b2a5fcf3b3405f3c44e/deepchem/utils/test/test_coordinate_box_utils.py#L85

Reply:
DONE!